cloud/ec2.go

```diff
@@ -1257,22 +1240,20 @@ func (m *ec2Manager) StopInstance(ctx context.Context, h *host.Host, user string
 		"distro":        h.Distro.Id,
 	})
 
-	return errors.Wrap(h.SetStopped(user), "failed to mark instance as stopped in db")
+	return errors.Wrap(h.SetStopped(user), "marking DB host as stopped")
 }
 
 // StartInstance starts a stopped EC2 instance.
 func (m *ec2Manager) StartInstance(ctx context.Context, h *host.Host, user string) error {
-	// Check that target instance is stopped
 	if h.Status != evergreen.HostStopped {
-		return errors.Errorf("cannot start '%s' - host is not stopped", h.Id)
+		return errors.Errorf("cannot start '%s' - host status is '%s'", h.Id, h.Status)
 	}
 
 	if err := m.client.Create(m.credentials, m.region); err != nil {
 		return errors.Wrap(err, "error creating client")
 	}
 	defer m.client.Close()
 
-	// Make request to start the instance
 	_, err := m.client.StartInstances(ctx, &ec2.StartInstancesInput{
```

Comment:
could it make sense to get the status from the response here like for stopping?

Reply:
The `GetInstanceInfo` is unavoidable here since we have to cache the instance data (e.g. to get the new IP address) in `cacheHostData`.